mlir/lib/CAPI/Dialect/MIGraphX.cpp

```diff
@@ -137,6 +137,10 @@ void mlirMIGraphXAddHighLevelPipeline(MlirPassManager pm) {
   passMan->setNesting(mlir::PassManager::Nesting::Implicit);
   mlir::migraphx::addHighLevelPipeline(*passMan);
   mlir::rock::buildBufferizePipeline(*passMan);
+  if (std::getenv("MIGRAPHX_MLIR_IR_PRINT")) {
```

Comment:
MIGraphX likes environment variables to control things.  Also, there's no place to hang --mlir-ir-print-all in a python script.  I needed this to see what rocMLIR was doing underneath python_3dunet.  enableIRPrinting() requires disableMultithreading().